docs-src/references/strongly-typed-errors/cause-unhandled-update.md

```diff
@@ -0,0 +1,17 @@
+---
+id: cause-unhandled-update
+title: Cause Unhandled Update
+description: Explanation for Cause Unhandled Update error message, and how to fix it.
+sidebar_label: Cause Unhandled Update
+tags:
+  - errors
+  - strongly-typed
+---
+
+`UnhandledUpdate` occurs when the [Workflow Execution](/concepts/what-is-a-workflow-execution) attempts to close itself before handling pending updates.
+
+This error can happen when the [Workflow](/concepts/what-is-a-workflow) is receiving a high number of [Signals](/application-development/features/#signals).
+Drain the Signal Channel with the `ReceiveAsync` function to prevent or resolve this error.
```

Comment:
sounds Go-specific, maybe say `In Go, ` and add a TODO comment to link to a new dev guide section on completing signal handling?

Reply:
Draining the signals doesn't prevent the error.
Are you sure this is referring to signals? What is "update" in this context?

Reply:
Getting info from @alexshtin and others as I go along.

Reply:
Not relevant, I'm resolving this conversation, this error is about workflow updates, not signals.